lib/bootleg/tasks.ex

```diff
@@ -90,6 +86,27 @@ defmodule Bootleg.Tasks do
     end)
   end
 
+  defp load_bootleg_tasks path do
```

Comment:
Is this a new style guide/code formatting syntax? We might consider following the same format we use elsewhere in the project, or updating it everywhere in a separate commit

Reply:
oops, oversight on my part, should have parens - especially now that with formatter it's The Elixir Way(TM)

Reply:
related: after this is merged we're gonna run `mix formatter` on the whole shebang

Reply:
also odd that credo didn't notice this inconsistency, hmmm